### ISSUE TITLE
(Feature Request) .sav file support 

### ISSUE THREAD

**Issue by spagootie** [steward-fu/nds#81]
I know the original DraStic supports saving with .sav files instead of .dsv files, it'd be very useful for syncing saves between devices.

**Comment by steward-fu** [comment]
Both .sav and .dsv are kept in my porting, please check in your microsd.

**Comment by spagootie** [comment]
How can I switch between saving as .sav and .dsv?

**Comment by vickyorlo** [comment]
While I can confirm that the port loads .sav files, it re-saves them as .dsv afterwards... It would be nice to have the option to use .sav files for comaptibility with flashcarts and other emulators.

**Comment by steward-fu** [comment]
In my understanding, the .sav file is used in game and the .dsv is used for quick save-state or load-state. You can copy the save file from the flash card and the overwritten the drastic's .sav file. After load in game, you can do 'quick save-state' in drastic emulator. Then, you can get the new .dsv file.

So, my question is: what is the meaning of  ".sav file support" ?

**Comment by vickyorlo** [comment]
Not quite. The savestate format for DraStic is .dss. The savestate format for DeSmuME is .dsx where x is the slot number.

.dsv is the DeSmuME battery save format. It is almost identical to the .sav battery save file format used by almost everything else, but it has a bunch of data appended at the end.

**Comment by steward-fu** [comment]
Thanks for your information. If so, I think you can use .sav file (from flashcard) directly on the DraStic emulator. Is my understanding correct ? If correct, then nothing needed to do for this issue, right ?

**Comment by steward-fu** [comment]
Thanks for your explanation. Both .sav and .dsv are in drastic folder. I mean user can copy it directly without any conversion. So, I am a little confusing why we need convert to .sav file ?

**Comment by SlyAceZeta** [comment]
They're saying that currently DraStic can *load* .sav files, but it cannot *save* .sav files.

**Comment by steward-fu** [comment]
The .sav file is for in-game state when playing NDS game. I mean if you do save the state in game, the .save file can be updated properly. So, my question is: do you need to update the .sav file automatically when exit NDS emulator ?
